mdata/aggregator.go

```diff
@@ -39,35 +39,39 @@ func NewAggregator(store Store, cachePusher cache.CachePusher, key schema.AMKey,
 		agg:  NewAggregation(),
 	}
 	for _, agg := range agg.AggregationMethod {
+		retentions := conf.Retentions{
+			Orig: "<autogenerated>",
```

Comment:
What's the point of setting this `Orig` string here?

Reply:
what I did here was very gross and i should be ashamed.
the point of setting an orig string is currently about non-existent as we know there is only 1 place where we actually do something with Orig, and that is for the render metadata, which doesn't use these values.
we just set it here in case somebody ever wants to use it. But if we try to accommodate future use cases, we should really just set it correctly. I've just pushed a patch to fix this.